tutorials/3d/portals/appendix.rst

```diff
@@ -0,0 +1,87 @@
+Appendix
+========
+Example SceneTree
+~~~~~~~~~~~~~~~~~
+Putting all the ideas together, here is an example scene tree:
+
+.. image:: img/example_scenetree.png
+
+- We have used a :ref:`RoomGroup<class_RoomGroup>` to denote an outside area
+- The :ref:`MeshInstance<class_MeshInstance>`\ s inside the :ref:`Room<class_Room>`\ s are either ``STATIC`` or ``DYNAMIC``
+- We have created a :ref:`Spatial<class_Spatial>` (I decided to call it 'Freeform', but you could use any name) under which to place ``STATIC`` and ``DYNAMIC`` objects that will be autoplaced in rooms
+  (Freeform is inside the ``roomlist``, but *not* inside a room).
+- The player and the monsters are on branches *OUTSIDE* the ``roomlist``
+- The player and monster meshes have portal mode ``ROAMING`` so they can be in any room
+- The camera is outside the ``roomlist``
+
+.. _doc_rooms_and_portals_blender:
+
+Creating room systems in Blender (or other modeling tools)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Although you can create your room system entirely within the editor, you can also build rooms and portals within your modeling tool. There is one small snag - modeling tools such as Blender have no knowledge of Room, RoomGroup and Portal nodes. In order to work around this we use the naming conventions that were discussed earlier.
+
+Rooms should be created as Empties with names such as ``Room_kitchen`` (substituting your preferred room name). If you place meshes within the rooms within the modeling tool, these Empties will form Spatial nodes in Godot, and will be converted to room nodes automatically during the room conversion phase.
+
+Creating a RoomGroup is easy - any ``Empty`` with the name prefix ``RoomGroup`` will be converted to a RoomGroup.
```

Comment:
```suggestion
Creating a RoomGroup is done by creating an Empty object with the name prefix ``RoomGroup``. For instance, you can create an Empty object called ``RoomGroup_house`` and it will be converted to a RoomGroup node when imported in Godot.
```